WeWatch/WeWatch/UICommons/LargeCover.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  LargeCover.swift
+//  WeWatch
+//
+//  Created by Anton on 27/12/2024.
+//
+
+import SwiftUI
+
+internal struct LargeCover: View {
```

Comment:
Although we’ve already discussed this, I still propose renaming the `LargeCover` component to `ImagePlaceholder`.